final_results/result_processing_wrapper.sh

```diff
@@ -36,21 +36,52 @@ else
   REGEX="java-1.8.0"
 fi
 
+graph_parameters() {
+##GRAPH_NAME="LOCAL" ##necessary for graph naming
+echo "running graph parameters"
+if [[ ($1 == *"local"*) ]];then
+  graph_name=local"$2"
+else
+  graph_name=virtual"$2"
+fi
+}
+
 for res in $RESULTS ; do
   echo $res
   echo $REGEX
   echo $benchmark
-  if [[ ($res == *"DACAPO"*) && ($benchmark == *"DACAPO"*)]];then
-    python $SCRIPT_DIR/result_processing.py "$res" "geom" "summary.txt" True $REGEX
+
+  if [[ ($res == *"DACAPO"*) && ($benchmark == *"DACAPO"*)]];then   
```

Comment:
it seems like some fomratting cnhage have leaked here. Please rever it. it cannot be reviewd like htis.